pwnlib/heap/glmalloc/malloc_chunk.py

```diff
@@ -0,0 +1,209 @@
+from pwnlib.heap.glmalloc.utils import p32, p64
```

Comment:
These should be imported from `pwnlib.util.packing`

Reply:
Done in 689f56e